packages/server/src/sw.ts

```diff
@@ -0,0 +1,154 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
```

Comment:
Maybe we could rename this file from `sw.ts` to `serviceworker.ts` so it's more explicit?

And the existing `serviceworker.ts` to `serviceworkerwrapper.ts`?

Reply:
perhaps `serviceworker.ts` and `servicemanager.ts`?

Reply:
3a39e7f3 went with `ServiceManager`, etc

Reply:
Thanks. Although `ServiceManager` can probably be a bit confusing since there is already a `ServiceManager` in JupyterLab?

Reply:
`ServiceWorkerManager` would be fine I guess?

Reply:
_Excellent_ point, updated in 0bf86c21.